process/subgroups.md

```diff
@@ -0,0 +1,69 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
+
+- An area of interest that needs a lot of discussion. Creating a subgroup with regular meetings provides more bandwidth for this discussion.
+- An area of interest is only relevant to a small group of participants. In these cases, subgroup meetings give a more focused venue. This way, participants don’t have concerns about wasting the CG’s time with something that the rest of the CG is not engaged in.
+
+To create a subgroup, you should add a poll to a CG meeting agenda with at least 7 days notice. As part of the poll, you should provide information about who will be chairing the subgroup.
+
+## Current Subgroups
+
+| Subgroup  | Chair(s)  | Discussion Repo  | Meetings  |
+|---|---|---|---|
+| Debugging  | Derek Schuff (@dschuff)  | [Repo](https://github.com/WebAssembly/debugging)  | no meetings currently scheduled, but discussion or agenda suggestions are welcome |
+| GC  | Thomas Lively (@tlively)  | [Repo](https://github.com/WebAssembly/gc)  | [Agendas](https://github.com/WebAssembly/gc/issues?q=is%3Aissue+%22Agenda+for+subgroup+meeting%22)  |
+| SIMD  | Petr Penzin (@penzn), Ng Zhi An (@ngzhian)  | ? | [Agendas](https://github.com/WebAssembly/meetings/tree/main/simd/)  |
+| Stack Switching  | Francis McCabe (@fgmccabe)  | [Repo](https://github.com/WebAssembly/stack-switching)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/stack)  |
+| WASI  | Lin Clark (@linclark), Sam Clegg (@sbc100)  | [Repo](https://github.com/WebAssembly/WASI)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/wasi)  |
+
+## Subgroup Leadership Expectations
+
+Subgroups require two kinds of leadership. These roles can be filled by the same person or multiple people:
+
+- Chair(s)—responsible for running the group
+- Champion(s)—responsible for driving technical progress on proposals
+ 
+### Chair
+A subgroup chair is expected to:
+
+- Organize subgroup meetings:
+  - Create agendas in advance so that others can add issues via PRs. These agendas should live in a subgroup-specific folder in the [WebAssembly meetings repo](https://github.com/WebAssembly/meetings). Alternatively, agendas can be organized via issues on a proposal repo for subgroups covering just a single proposal.
```

Comment:
I wouldn't mind if we dropped the second approach (organising agendas through issues) and switched all subgroups to using agenda docs properly. That's vastly better in terms of discoverability, transparency, and organisational convenience. Finding out about an upcoming meeting's agenda by having to (a) search for the right issue, and (b) read the entire discussion to extrapolate the actual agenda is a bad user experience.

Reply:
That seems fair. I commented on the GC subgroup's latest agenda issue proposing a switch to the new system after the upcoming meeting. https://github.com/WebAssembly/gc/issues/268. Assuming no one objects to that in the GC subgroup, I would be happy to remove this text here.

Reply:
From @penzn comment above, it looks like the SIMD group could move to the using the agenda docs as well. @fgmccabe is this something you would be interested in for the stack-switching meetings? Resolving this seems independent to this document though, as it's documenting how the meetings are currently run. As the process changes, it can be reflected in this document as well.

Reply:
One point is that issues are still searchable because Github shows both issues and repo contents in search. However if somebody is looking for past meeting agendas, they would have to go through both project repo and meetings repo.